schemas/context/person.example.1.json

```diff
@@ -5,5 +5,8 @@
     "xdm:lastName": "Doe",
     "xdm:fullName": "Jane F. Doe"
   },
-  "xdm:birthDate": "1996-01-03"
+  "xdm:birthDate": "1996-01-03",
+  "xdm:gender": "female",
+  "xdm:maritalStatus": "single",
+  "xdm:nationality": "CAN"
```

Comment:
Match it with the specifications, [A-Z]{2}

Reply:
Fixed in new push